test/test_transforms.py

```diff
@@ -448,23 +459,23 @@ def test_pad_with_non_constant_padding_modes(self):
         # First 6 elements of leftmost edge in the middle of the image, values are in order:
         # edge_pad, edge_pad, edge_pad, constant_pad, constant value added to leftmost edge, 0
         edge_middle_slice = np.asarray(edge_padded_img).transpose(2, 0, 1)[0][17][:6]
-        self.assertTrue(np.all(edge_middle_slice == np.asarray([200, 200, 200, 200, 1, 0])))
+        assert_equal(edge_middle_slice, np.asarray([200, 200, 200, 200, 1, 0]), check_dtype=False, check_stride=False)
```

Comment:
What are the dtypes here? IMO we should only use `check_dtype=False` in case we don't care about the dtype. That is not the case here. `F.pad` shouldn't change the dtype, does it?